aws-integration/trace-sample-app/values.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    io.kompose.service: sample-app
+  name: sample-app
+spec:
+  ports:
+    - name: "4567"
+      port: 4567
+      targetPort: 4567
+  selector:
+    io.kompose.service: sample-app
+status:
+  loadBalancer: {}
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    io.kompose.service: sample-app
+  name: sample-app
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      io.kompose.service: sample-app
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+      labels:
+        io.kompose.service: sample-app
+    spec:
+      containers:
+        - env:
+            - name: AWS_REGION
+              value: ${AWS_REGION}
```

Comment:
Why do we need the region here? Such things should be configured in the otel collector.